tox.ini

```diff
@@ -1,8 +1,8 @@
 [tox]
 envlist = py
-isolated_build = true
```

Comment:
what prevents using isolated builds? (what does the `isolated_build` parameter do in the first place?)

Reply:
isolated_build was removed in tox4, the package is always built in a separate venv